dbt_copilot_helper/templates/addons/svc/subscription-filter.yml

```diff
@@ -0,0 +1,21 @@
+# {% version_info %}
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: The name of the service, job, or workflow being deployed.
+Resources:
+  SubscriptionFilter:
+    Type: AWS::Logs::SubscriptionFilter
+    Properties:
+      RoleArn: !Sub "arn:aws:iam::${AWS::AccountId}:role/CWLtoSubscriptionFilterRole"
+      LogGroupName: !Sub "/copilot/${App}/${Env}/${Name}"
+      FilterName: !Sub "${App}-${Env}-stream"
+      FilterPattern: ""
+      # SubscriptionFilter requires the Destination ARN of the CloudWatch Logs in the main archive logs account.
+      DestinationArn: "arn:aws:logs:eu-west-2:812359060647:destination:cwl_log_destination"
```

Comment:
I'm wondering if we can parameterise the account number so it's not part of the library?

Reply:
we have the same issue with the appconfig-ipfilter.yml.  we have the arn in the code.  Maybe something we pick up seperately for all files

Reply:
OK, happy to approve this if you can make a ticket to look at solving that.